### PR TITLE
Mobile devices

### PR DESCRIPTION
**Pull request by deadlight**
## Done

New mobile devices page and associated styling

## QA

Check `/mobile/devices` in all viewports and check it against the [copydoc](https://docs.google.com/document/d/1l_P2yEVeF6tIvSsm7LAO9ds9nf_ZenKhwlOe3xZybBc/edit#)

### CODE REVIEW

File: templates/mobile/shared/_devices_newsletter.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<div id="mc_embed_signup" class="box-digest promo-signup">
```
Comment (nottrobin) [inline templates/mobile/shared/_devices_newsletter.html]: I assume other things is gonna use this include?

Comment (deadlight) [inline templates/mobile/shared/_devices_newsletter.html]: yes

**Comment by nottrobin** [comment]
## Differences from copy docs

Please update either markup or copy doc:

- "BQ Aquaris M10 Ubuntu Edition" - copy doc has "tablet" on the end
- Copy code has no description under this ^ heading
- s/Micro HDMI port/Micro HDMI slot/
- "Alternatively, flash your own device" is a H2 in copy doc, H3 in page
- Under this heading ^ copy doc says "Find out more >", markup says "Find out how to install Ubuntu on a mobile device" (Notice missing chevron as well as different text)
- Under "Did you know Ubuntu is available on IoT devices too", missing link to Ubuntu Core

## Newsletter sign-up

- No error message on error, just red box around element. Is that expected?
- Confirmation message looks unstyled:

confirmation message
small screen

File: templates/mobile/devices.html [hunk 2]
```diff
@@ -4,19 +4,182 @@
...
 {% block content %}
 
-[DEVICES]
+<section class="row row-hero">
+    <div class="strip-inner-wrapper">
+        <div class="six-col">
+            <h1>Mobile devices</h1>
+            <p>The Ubuntu experience is available on a number of commercial devices. </p>
+            <ul class="list-ubuntu">
+                <li>Mobile devices that  turn into PCs*</li>
```
Comment (nottrobin) [inline templates/mobile/devices.html]: Shouldn't the `*` here be superscript or something?

Comment (deadlight) [inline templates/mobile/devices.html]: We explored options and decided on normal text styling.

**Comment by deadlight** [comment]
- "BQ Aquaris M10 Ubuntu Edition" - copy doc has "tablet" on the end - **fixed**
 - Copy code has no description under this ^ heading - **removed**
 - s/Micro HDMI port/Micro HDMI slot/ - **fixed**
 - "Alternatively, flash your own device" is a H2 in copy doc, H3 in page - **this is intentional, copydoc doesn't reflect the structure perfectly but I've changed the copydoc to avoid confusion**
 - Under this heading ^ copy doc says "Find out more >", markup says "Find out how to install Ubuntu on a mobile device" (Notice missing chevron as well as different text) - **I updated the copydoc and the chevron was erroneous (external links never have one)**
 - Under "Did you know Ubuntu is available on IoT devices too", missing link to Ubuntu Core - **done**

Newsletter fixes to come.

**Comment by deadlight** [comment]
@nottrobin Newsletter form feedback cleaned up. I'll put in an issue to look at how we style those, globally.

**Pull request merged**